man/jail.conf.5

```diff
@@ -146,6 +146,12 @@ Ensure syslog or the program that generates the log file isn't configured to com
 .B logencoding
 encoding of log files used for decoding. Default value of "auto" uses current system locale.
 .TP
+.B banaction
+default banning action (iptables-multiport) for all jails specified in the \fI[DEFAULT]\fR section.
```

Comment:
Yes. But my comment is about documentation.  Now it is misleading and needs to be fixed up.

On October 31, 2015 11:08:46 AM EDT, "Serg G. Brester" <notifications@github.com> wrote:
>> @@ -146,6 +146,12 @@ Ensure syslog or the program that generates the
>log file isn't configured to com
>>  .B logencoding
>>  encoding of log files used for decoding. Default value of "auto"
>uses current system locale.
>>  .TP
>> +.B banaction
>> +default banning action (iptables-multiport) for all jails specified
>in the \fI[DEFAULT]\fR section.
>
>Yes I know, but point of this commit resp. of this variables to specify
>it central in [DEFAULT] section of `jail.local`, just to not specify it
>per jail.
>Also one place for all "normal" jails and one for "allports" jails...
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/fail2ban/fail2ban/pull/1238/files#r43572728